packages/sls-aws/src/eventbridge/json-eventbridge-message.ts

```diff
@@ -0,0 +1,16 @@
+import type { Context, EventBridgeEvent } from 'aws-lambda'
+
+import { PayloadError, PayloadParseError } from '../error/parse'
+
+const jsonEventBridgeMessage = <T>(event: EventBridgeEvent<string, string>, _context: Context): T => {
```

Comment:
I don't believe the `event.detail` is actually stringified JSON https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-events.html

Also, within product-import-submit we use things like the `id` from the EventBridge event:
https://github.com/boohoo-com/debenhams/blob/f43a5001e2349f6fd49b6cb7d6c84fcbd54032f0/services/product-import-submit/src/common/replay-event.ts#L34

So would we need to essentially return the entire event bridge event?

Reply:
Hi, certainly in our case the detail is a stringified JSON (I'll send you an example message). But you're right, in the SDK the type is TDetail (I thought this just extended string, but it might wrong), so that wouldn't cover all cases.

As for the id, yes we use that to create the replay message so we need the whole event bridge event.